spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -1998,8 +2007,9 @@ def _recover_from_error(self, exception, exc_info, executable_targets):
                 failed_cores = self._txrx.get_cores_not_in_state(
                     self._executable_types[executable_type],
                     executable_type.end_state)
-                for (x, y, p), core_info in failed_cores:
-                    unsuccessful_cores.add_processor(x, y, p, core_info)
+                for (x, y, p) in failed_cores:
+                    unsuccessful_cores.add_processor(
+                        x, y, p, failed_cores.get_cpu_info(x, y, p))
```

Comment:
Won't this be slower than what it's replacing, given that get_cores_not_in_state(...) does this work for you already?

Reply:
the issue here is some change (which might have been fixed in master, but this worked) where the CoreInfo or CoreSubsets (which ever was replaced) doesnt have the right api and data to give us what we need for the failure state (mainly iobuf extractor)